src/main/java/training/busboard/web/BusInfo.java

```diff
@@ -0,0 +1,19 @@
+package training.busboard.web;
+
+public class BusInfo {
+    private final String postcode;
+    private String[] linesOfBusETA;

```

Comment:
It makes more sense to me to have this model store the buses themselves rather than this array of strings.